### DISCUSSION TITLE
[업적] 업적 디자인을 골라주세요!

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#232] [General]
### 사건의 발단
검증된 사용자를 위한 온도를 구현하기로 하였는데, 이는 현재 구현이 불가능 
왜🤔? 기준도 모호하고, 페이지도 다시 짜야함 ㅠㅠ

그래서 대신 업적으로라도 구현했으면 좋겠다 라고 의견을 내었고, 아까 다들 식사하러 가셨을 때 유현님과  이야기해본 결과, 업적은 서버통신을 빼고 User 객체에서 있는 값을 활용하면 구현의 난이도가 높지 않아 가능하다는 결론이 났습니다!
그리고 제가 계속 하자고 조른 부분이라 제가 디자인을 해보기로 했습니다. 
color는 남경님이 뽑아주신 대표컬러들을 가져다썼습니다 (남경님 항상 감사합니다)

### 현재 업적 가능한 것 
댓글
좋아요 
팔로우 언팔로우
게시글 생성
-> 이 모두 User 객체에 있는 내용이라 구현이 그닥 어렵지는 않음!

 
그래서 디자인을 여러개 뽑아봤는데 봐주시면 감사하겠습니당

참고한 자료는 이렇습니다. 이런 느낌으로 하고 hover하면 툴팁을 이용하여 해당 내용을 띄울 예정입니다 

<img width="363" alt="스크린샷 2022-06-20 오후 7 58 22" src="https://user-images.githubusercontent.com/72402747/174601827-c5c6585b-6f93-4d1d-aa6a-9bf82e135988.png">

1. 글자없이 아이콘만 - 남경님이 만들어 주신 로고처럼 border 있게 

2. 글자없이 아이콘만 - border 없게

3. 글자 + 아이콘 

<img width="637" alt="스크린샷 2022-06-20 오후 9 32 39" src="https://user-images.githubusercontent.com/72402747/174602438-6d20cd24-5d00-4b0f-b323-ee9924394b93.png">

### 현재 생각한 4가지 업적에 대해 명칭도 나름 붙여봤습니다.. ㅋㅋ 부끄럽군요,, 
다시 한 번 남경님의 디자인 실력에 감탄한 시간이었습니다.. 

<img width="470" alt="스크린샷 2022-06-20 오후 9 37 30" src="https://user-images.githubusercontent.com/72402747/174603240-238fd981-7826-472a-9430-c108d1e9c1d2.png">

### active 안 되었을 때

<img width="341" alt="스크린샷 2022-06-20 오후 10 23 54" src="https://user-images.githubusercontent.com/72402747/174611428-7ee211ac-3334-4f24-bd7f-5cd042a5e186.png">

**Comment by YuHyun-P** [comment]
저는 3번 네모 카드 + 탭으로 들어가기!

**Comment by YuHyun-P** [comment]
형진님 3번

**Comment by NamgyungKim** [comment]
저는 위에 스샷이요~
